### PR TITLE
feat(egeo): update to angular 8, include angular elements generator

### PR DESCRIPTION
**Pull request by jdiaz-stratio**
### Documentation
- [x] Add the issue in PR description
- [x] Have changelog updated
- [ ] You fill the milestone value
- [x] You add some label
- [ ] Have example running in demo app
- [ ] Review id on demo is the same of egeo folder

### Code
- [x] Pass Sass lint task
- [ ] Have qaTags

### Tests
- [ ] Have at least 70% tests coverage

### CODE REVIEW

File: projects/egeo-elements/compileHelpers.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const {execSync} = require('child_process');
+
+compileMainTheme();
+
+function compileMainTheme() {
+    const pathFrom = `../egeo/src`;
+    const pathTo = `dist/helpers`;
+
+    //execSync(`lessc ${pathFrom}/global-vars.less ${pathTo}/main-theme.css`);
```
Comment (emgaitan-stratio) [inline projects/egeo-elements/compileHelpers.js]: Remove commented line

File: projects/egeo-elements/compileElements.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
```
Comment (emgaitan-stratio) [inline projects/egeo-elements/compileElements.js]: Rename file to compile-elements.js

File: projects/egeo-elements/compileHelpers.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const {execSync} = require('child_process');
```
Comment (emgaitan-stratio) [inline projects/egeo-elements/compileHelpers.js]: Rename file to compile-helpers.js

File: projects/egeo/src/theme/global-vars.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+:root{
```
Comment (emgaitan-stratio) [inline projects/egeo/src/theme/global-vars.scss]: Why do yo need add this?

Comment (jdiaz-stratio) [reply projects/egeo/src/theme/global-vars.scss]: para usar variables de css desde los angular elements

File: projects/egeo-elements/compileHelpers.js (hunk 3 above)
Comment (jdiaz-stratio) [reply projects/egeo-elements/compileHelpers.js]: done

File: projects/egeo-elements/compileElements.js (hunk 2 above)
Comment (jdiaz-stratio) [reply projects/egeo-elements/compileElements.js]: done

File: projects/egeo-elements/compileHelpers.js (hunk 1 above)
Comment (jdiaz-stratio) [reply projects/egeo-elements/compileHelpers.js]: done

**Pull request merged**